config/chains/mainnet.exs

```diff
@@ -25,3 +25,5 @@ config :godwoken_explorer,
   },
   init_godwoken_l1_block_number: 5_744_446,
   ckb_token_script_hash: "0x0000000000000000000000000000000000000000000000000000000000000000"
+
+config :godwoken_explorer, :sourcify, chain_id: "71402"
```

Comment:
chain id of mainnet v0 is `71394`
Ref: https://github.com/nervosnetwork/godwoken-info/tree/main/mainnet_v0

Or we could get chain id by RPC
```
curl -X POST 'https://v1.mainnet.godwoken.io/rpc' \
  -H "Content-Type: application/json" \
  -d '{"jsonrpc":"2.0","method":"eth_chainId","params": [],"id":1}'
```

Reply:
use ENV variables to config it